modules/mini-games/Race.lua

```diff
@@ -477,6 +483,16 @@ Gui.element{
     tooltip = 'Fuel'
 }
 
+--- Used to select what map to play on
+-- @element map_dropdown
+local map_dropdown=
+Gui.element{
+    type = 'drop-down',
+    items = {config[1].name, config[2].name},
```

Comment:
Would be better to make this more dynamic and create a local var called `map_names` which was populated when the file loaded by looping over all the configs. Speedrun does this using the `targets` table.

Reply:
their will be no new maps so i dont see why